orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/utils/TrafficGuard.java

```diff
@@ -53,6 +61,7 @@ public void verifyInstanceTermination(String serverGroupNameFromStage,
                                         Location location,
                                         String cloudProvider,
                                         String operationDescriptor) {
+    // TODO rz - Expose a single `instance server groups` endpoint in clouddriver; returns a map<instanceid, servergroup>.
```

Comment:
do you mean `map<servergroup, List<instanceId>>` like this seems to compute?

Reply:
Ha yes.